Content.Shared/_Impstation/ClothingMobStateVisuals/ClothingMobStateVisualsComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._Impstation.ClothingMobStateVisuals;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class ClothingMobStateVisualsComponent : Component
```

Comment:
```suggestion
public sealed partial class ClothingBasedMobStateVisualsComponent : Component
```
small rename to make what this does a little more clear

Reply:
I disagree, I think this actually makes it *less* clear what the component is doing. "Clothing-Based MobState Visuals" implies to me that the mob's visuals are what's changing, when in reality it's the clothing's visuals that are changing.

Reply:
I have instead renamed it to MobStateClothingVisuals.